### PR TITLE
Added author info for blogs

### PR DESCRIPTION
**Pull request by linkfang**
Signed-off-by: Zhou Fang <zhou.fang@eclipse-foundation.org>

Fixes https://github.com/EclipseFdn/ecdtools.eclipse.org/issues/213
https://github.com/EclipseFdn/ecdtools.eclipse.org/issues/213#issuecomment-1067261961

### CODE REVIEW

File: content/resources/blogs/2021-12-16-theia-i18n/index.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ date = "2021-12-16"
 title = "Internationalization in Eclipse Theia"
 summary = "Even though some attempts by contributors have been made in the past, as of mid 2021 still Theia had no support for different locales. Arduino has addressed this with  the all new Arduino IDE based on Theia. Having supported Arduino previously with their IDE, they tasked TypeFox with contributing this feature to Theia."
 category = "blog"
+author = "Brian King"
+email = "briks.si@gmail.com"
```
Comment (autumnfound) [inline content/resources/blogs/2021-12-16-theia-i18n/index.md:7]: We should be careful attributing this blog to Brian, as the first byline states it was written by someone else

Comment (linkfang) [reply content/resources/blogs/2021-12-16-theia-i18n/index.md:7]: Good catch! Will confirm with Clark

Comment (linkfang) [reply content/resources/blogs/2021-12-16-theia-i18n/index.md]: Updated to Mark Sujew

**Comment by linkfang** [comment]
The author info updates can be previewed here:
https://deploy-preview-230--eclipsefdn-ecdtools.netlify.app/index.xml

And it's still a valid RSS feed:
https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fdeploy-preview-230--eclipsefdn-ecdtools.netlify.app%2Findex.xml

Ready to be merged

**Pull request merged**